Models/CurrentPrice.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Threading.Tasks;
+using FoodKept.Models;
+
+namespace FoodKept.ViewModels
+{
+    public class CurrentPrice
+    {
+        public CurrentPrice() { }
```

Comment:
why it is needed if it's empty

Reply:
This was used to initialize the object before. Now the constructor is not needed. Will fix